manifests/params.pp

```diff
@@ -441,7 +441,7 @@
   $server_idle_timeout                    = 1200000
   $server_web_idle_timeout                = 30000
   $server_connect_timeout                 = 120000
-  $server_enable_ruby_profiler            = false
+  $server_enable_ruby_profiler            = true
```

Comment:
Excuse the ignorance, it's been a while :P - is `$server_enable_ruby_profiler` used for anything else at this point? If it's just for debugging and such, then it's probably OK to drop it, I think.